core/frontend/src/components/parameter-editor/ParameterEditor.vue

```diff
@@ -56,16 +56,20 @@
           <v-container
             v-if="edited_param"
           >
-            <v-row v-if="edited_param.range">
-              Max: {{ edited_param.range.high }}
-              Min: {{ edited_param.range.low }}
-            </v-row>
             <v-row v-if="edited_param.rebootRequired">
               Reboot Required
             </v-row>
             <v-row>
               {{ edited_param.description }}
             </v-row>
+            <v-row
+              v-if="edited_param.range"
+              class="pt-6"
+            >
+              Min: {{ edited_param.range.low }}
+              Max: {{ edited_param.range.high }}
+              Increment: {{ edited_param.increment ?? 1 }}
```

Comment:
I don't think we should assume the increment is 1.
I'd rather just omit it if it is not set.

Imagine a situation where the range is smaller than 1. That would be awkward.

we could make also make the input step be (high-low)/100 in case increment isnt set

Reply:
That's already our default behaviour, changing to 0.01.

Reply:
Done.